cli/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,11 @@
+# http://docs.aws.amazon.com/lambda/latest/dg/java-logging.html#java-wt-logging-using-log4j
```

Comment:
To the best of my knowledge, this file should only impact the CLI.. Right?

Reply:
Correct.